src/ucp/core/ucp_mm.c

```diff
@@ -1362,18 +1384,11 @@ void ucp_mem_rcache_cleanup(ucp_context_h context)
     }
 }
 
-ucs_status_t ucp_mem_reg_md_map_update(ucp_context_h context)
+ucs_status_t ucp_mm_init_alloc_md_index(ucp_context_h context)
```

Comment:
this func could be moved to context.c and be static

Reply:
This function uses static `ucp_memh_alloc`. `ucp_memh_alloc` should be moved to .h file to do this. I'd rather keep `ucp_mm_init_alloc_md_index` in .h file.

Reply:
It also requires to include `ucp_mm.inl`. Which can be removed from context.c actually.